core/src/main/resources/hudson/model/ModifiableViewGroup/_api.jelly

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    <h2>Create View</h2>
+    <p>
+        To create a new view, post <code>config.xml</code> to <a href="../createView">this URL</a> with
```

Comment:
avoid using here or other similar concepts in links, see https://uxmovement.com/content/why-your-links-should-never-say-click-here/

possibly should also have `<code>` wrapped around it
```suggestion
        To create a new view, post <code>config.xml</code> to <a href="../createView">createView</a> with
```